### PR TITLE
Ifpack2: fix bug when import/export not needed to combine overlap when performing block relaxation in serial

### PR DESCRIPTION
**Pull request by rstumin**
Auto-PR for SHA 6063060

### CODE REVIEW

File: packages/ifpack2/src/Ifpack2_BlockRelaxation_def.hpp [hunk 1]
```diff
@@ -701,13 +701,15 @@ initialize ()
...
         typedef Tpetra::MultiVector<        typename MatrixType::scalar_type, typename MatrixType::local_ordinal_type,  typename MatrixType::global_ordinal_type,typename MatrixType::node_type> scMV;
         import_type theImport = A_->getGraph()->getImporter();
-        scMV nonOverLapW(theImport->getSourceMap(), 1, false);
-        Teuchos::ArrayRCP<scalar_type > w_ptr = W_->getDataNonConst(0);
-        Teuchos::ArrayRCP<scalar_type> nonOverLapWArray = nonOverLapW.getDataNonConst(0);
-        nonOverLapW.putScalar(STS::zero ());
-        for (int ii = 0; ii < (int) theImport->getSourceMap()->getLocalNumElements(); ii++)  nonOverLapWArray[ii] = w_ptr[ii];
-        nonOverLapW.doExport (*W_,         *theImport, Tpetra::ADD);
-        W_->doImport(         nonOverLapW, *theImport, Tpetra::INSERT);
+        if (theImport.getRawPtr() != 0) {
```
Comment (cgcgcg) [inline packages/ifpack2/src/Ifpack2_BlockRelaxation_def.hpp:704]: `! theImport.is_null()`?

File: packages/muelu/test/scaling/CMakeLists.txt [hunk 2]
```diff
@@ -288,6 +288,14 @@ IF (${PACKAGE_NAME}_HAVE_TPETRA_SOLVER_STACK)
     NUM_MPI_PROCS 4
     COMM mpi # HAVE_MPI required
   )
+  TRIBITS_ADD_TEST(
+    Driver
+    NAME "GeneralBlockSmoothing"
```
Comment (GrahamBenHarper) [inline packages/muelu/test/scaling/CMakeLists.txt:293]: Won't tribits complain if this test has the same name as the test above it?

**Review by GrahamBenHarper — APPROVED** [review]
Just one comment about the test name

File: packages/ifpack2/src/Ifpack2_BlockRelaxation_def.hpp (hunk 1 above)
Comment (rstumin) [reply packages/ifpack2/src/Ifpack2_BlockRelaxation_def.hpp]: That does look nicer. I've revised.

File: packages/muelu/test/scaling/CMakeLists.txt (hunk 2 above)
Comment (rstumin) [reply packages/muelu/test/scaling/CMakeLists.txt]: Not sure ... but it is probably better to change the name which I just did by prepending the word Serial to the name.

**Review by GrahamBenHarper — APPROVED** [review]
Now it's in the AT's hands :)

**Pull request merged**